PPTP-1648: Added page

mauro-fanton opened the pull request:
hookup the change link and added more test

### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [X] `./precheck` was executed (Integration/Component/Unit tests)
 - [X] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added - N/A
 - [ ] Links to dependencies have been included (BE/FE work) - N/A
 - [X] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave -N/A

PSouthern reviewed (commented):
Do the migrate files need to be in here?

mauro-fanton commented:
Yes, the migrate file is to do with the scaffolding. I just changed a little to allow to accept a parameter to allow to create route using snake case format. As default this create routes using Camel Case format.

mauro-fanton commented:
> Yes, the migrate file is to do with the scaffolding. I just changed a little to allow to accept a parameter to allow to create route using snake case format. As default this create routes using Camel Case format.

Actually I reverted the migrate.sh changes, as this should not gone in this PR. This should go inn a separate changes by itself so it can be discussed if it worth it doing.

liampan commented on app/models/UnitMeasureType.scala:
```diff
@@ -0,0 +1,25 @@
+
+package models
+
+object UnitMeasureType extends Enumeration {
```
as discussed, lets use the viewmodels converter

liampan commented on .gitignore:
```diff
@@ -35,3 +35,4 @@ test-result
 tmp
 .idea_modules
 bin
+*.bak
```
bark 🐶

mauro-fanton replied:
removed

liampan commented on app/viewmodels/checkAnswers/HumanMedicinesPlasticPackagingWeightSummary.scala:
```diff
@@ -24,7 +24,7 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.implicits._
 
-object HumanMedicinesPlasticPackagingWeightSummary extends SummaryViewModel {
+class HumanMedicinesPlasticPackagingWeightSummary extends SummaryViewModel {
```
as discussed, lets turn these back to objects

mauro-fanton replied:
done

liampan commented on app/viewmodels/checkAnswers/ImportedPlasticPackagingSummary.scala:
```diff
@@ -24,14 +24,14 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.implicits._
 
-object ImportedPlasticPackagingSummary extends SummaryViewModel {
+class ImportedPlasticPackagingSummary(key: String ) extends SummaryViewModel {
```
as discussed, lets turn these to private classes, with static objects in the companion

mauro-fanton replied:
done

liampan reviewed (changes requested):
Looks good! few comments we've discussed :)
